src/components/Book.js

```diff
@@ -0,0 +1,50 @@
+const Book = () => (
+  <>
+    <div className="card">
+      <div className="title">
+        <span>Action</span>
+        <h1>Kung Fu Panda: The Dragon Knight</h1>
+        <p>Unseen Content</p>
+        <ul>
```

Comment:
- [ ] Great job 👏 . Please, create a prop for the book component and it should render the title and author of the book as a prop. 🙏